lib/index.js

```diff
@@ -687,6 +687,8 @@ class GalleryVideoArticle extends BaseAsset {
         super();
         this._object_type = "ArticleObject";
         this._content_type = "text/html";
+
+        console.warn('GalleryVideoArticle is deprecated. Use VideoArticle instead.');
```

Comment:
Do we unconditionally want to switch to `VideoArticle` in all cases?

Reply:
I'm not sure about doing that just yet - I'd prefer to test each one rather than forcing a switch.

Reply:
I meant the other way around — we don't necessarily want a deprecation message if there are still legitimate uses of `GalleryVideoArticle`.

Reply:
Hmm. I would like to warn future users against using it, since generally speaking, I think we want to migrate away from it in all cases. The concern I had was about migrating away from it piecemeal as opposed to forcing a potentially breaking change.